api/handlers.go

```diff
@@ -17,6 +17,18 @@ import (
 	"gitlab.com/NebulousLabs/errors"
 )
 
+// healthHandler returns the status of the service
+func (api *API) healthHandler(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	status := struct {
+		DBAlive bool `json:"dbAlive"`
+	}{}
+	_, err := api.staticDB.UserBySub(req.Context(), "", false)
```

Comment:
I believe mongo clients have a `Ping` method you could use to accomplish this. This will work as well though so I'll approve the PR. Just saying that `Ping` is probably a good option as well for a health check.

Reply:
Good point. F/U: https://github.com/SkynetLabs/skynet-accounts/pull/53